backend/routes/templates.go

```diff
@@ -99,7 +100,22 @@ func addTemplateImg(w http.ResponseWriter, r *http.Request) {
 
 	imageData := imageToPixelData(fileBytes)
 	hash := hashTemplateImage(imageData)
-	// TODO: Store image hash and pixel data in postgres database
+	templateId := 0
+	err = core.ArtPeaceBackend.Databases.Postgres.QueryRow(context.Background(), "INSERT INTO TemplateData (hash, data) VALUES ($1, $2) RETURNING key", hash, imageData).Scan(&templateId)
+	if err != nil {
+		WriteErrorJson(w, http.StatusInternalServerError, "Failed to insert template data in postgres")
+		return
+	}
+	// Set pixel in postgres
+	name := ""
+	position := 0
+	reward := 0
+	rewardToken := "RWRD"
+	_, err = core.ArtPeaceBackend.Databases.Postgres.Exec(context.Background(), "INSERT INTO Templates (name, width, height, position, reward, reward_token, template_id) VALUES ($1, $2, $3, $4, $5, $6, $7)", name, width, height, position, reward, rewardToken, templateId)
```

Comment:
It would be best to only update the `TemplateData` table here, so this can be removed. The `Templates` table will be managed by the indexer.

Reply:
Ok, insert in `Templates` has been removed.